src/frontend/handlers.go

```diff
@@ -199,6 +209,9 @@ func (fe *frontendServer) productHandler(w http.ResponseWriter, r *http.Request)
 		"platform_css":    plat.css,
 		"platform_name":   plat.provider,
 		"is_cymbal_brand": isCymbalBrand,
+		"cluster":         deploymentDetails[CLUSTERNAME],
+		"zone":            deploymentDetails[ZONE],
+		"pod":             deploymentDetails[HOSTNAME],
```

Comment:
Since `getDeploymentDetails(r)` returns a `map`, I would consider simply sub'ing this for:
```
"deploymentDetails": getDeploymentDetails(r)
```

That would reduce the delta of new lines from this PR by half 😄 

I would also remove the 3-4 vars at the top, they're fairly overkill for this application.

Both of these changes would adhere a bit better to DRY and KISS concepts.

Thoughts?

Reply:
My thought was to keep the templating _(in html)_ simple and everything evident from the code itself. Directly returning the map is makes the templating a bit confusing and also set precedence for people to keep adding to this map in future. I think a bit of verbosity makes it easy for someone new to grasp the codebase faster.

In any case I don;t mind changing it for simplicity. So I adopted the suggestions

Reply:
Changes in https://github.com/GoogleCloudPlatform/microservices-demo/pull/636/commits/cad4cdb35cb507f2580b67e82f12264a3154ac47

Reply:
For what it's worth, I +1 to Olivier's suggestion.